src/Makefile.am

```diff
@@ -389,10 +389,10 @@ endif
 
 bitcoind_LDADD = \
   $(LIBBITCOIN_SERVER) \
+  $(LIBBITCOIN_WALLET) \
```

Comment:
Why this change? Does the order matter? 🙄

Reply:
Yes, the order matters